dashboard2/publishwebsite.md

```diff
@@ -24,57 +24,45 @@ This screen opens by default after clicking on a _Target locale_ in your list. D
 
 - *Override HTML lang attribute*: Every HTML page has an attribute that specifies the language of the document. By default, this is set to the locale code (like `en-GB`). It's advised by the [W3](https://www.w3.org/International/articles/language-tags/) to "keep the tag as short as possible. Avoid region, script or other subtags except where they add useful distinguishing information". To follow this advice, you can override the default using this field.
  
-There are 3 tabs full of additional settings available as well:
+There are 3 tabs – ACCESS CONTROL, PUBLISH WIZARD and SETTINGS – full of additional settings available as well:
```

Comment:
I'm a big proponent of the _Oxford Comma_, and would like to see it used in the docs consistently.